mitxpro/utils.py

```diff
@@ -646,3 +646,52 @@ def strip_datetime(date_str, date_format, date_timezone=None):
 
     date_timezone = date_timezone if date_timezone else datetime.UTC
     return datetime.datetime.strptime(date_str, date_format).astimezone(date_timezone)
+
+
+def get_courserun_date_errors(
```

Comment:
A better name would be `validate_courserun_dates`. It can return a tuple of `True/False, Message`